### PR TITLE
[SimCode] refactor simVarString

### PR DESCRIPTION
**Pull request by phannebohm**
I couldn't understand what that function did so I rewrote it.

### CODE REVIEW

File: OMCompiler/Compiler/SimCode/SimCodeUtil.mo [hunk 1]
```diff
@@ -8228,55 +8228,32 @@ end derVarFromStateVar;
 public function simVarString
```
Comment (lochel) [inline OMCompiler/Compiler/SimCode/SimCodeUtil.mo:8228]: We should have a short doc string that explains the purpose of the function.

**Review by lochel — COMMENTED** [review]
I appreciate your effort in making the codebase a little more readable. 👍

File: OMCompiler/Compiler/SimCode/SimCodeUtil.mo (hunk 1 above)
Comment (phannebohm) [reply OMCompiler/Compiler/SimCode/SimCodeUtil.mo:8228]: Absolutely!

**Comment by phannebohm** [comment]
I forgot that i also changed the dump so that
`index: 0: $CLKPRE.x1 (no alias)  initial: x1no arrCref index:(1) [10]`
is now
`0: $CLKPRE.x1 initial: x1 index:(1) [10]`
Is that too concise?

Also I forgot how to update the testsuite. I tried `./runtest.pl path/to/model.mos -b` but it complained about uninitialized values.

**Pull request merged**